.prettierrc

```diff
@@ -1,4 +1,5 @@
 {
-    "tabWidth": 4,
-    "useTabs": true
-}
\ No newline at end of file
+	"tabWidth": 4,
```

Comment:
Idk what youre VS Code done here, but the complete codebase uses tabWith 8 in your PR.